private/api/classrooms/update-classroom.php

```diff
@@ -0,0 +1,35 @@
+<?php
+require_once('../db-connection/root-connection.php');
+try{
```

Comment:
we need to implement validation of input so i matches DB-types. Maybe create a TODO:  ? :)

Reply:
will do